components/calendar/src/provider.rs

```diff
@@ -71,8 +71,8 @@ icu_provider::data_marker!(
 );
 icu_provider::data_marker!(
     /// Week information
-    CalendarWeekV2,
-    "calendar/week/v2",
+    CalendarWeekV1,
```

Comment:
I believe @sffc didn't want to re-v1 keys, unless we have never published an icu4x with this v1

I somewhat disagree, we broke a large fraction of data via the VarZeroVec changes anyway, it's not worth it to maintain the data key versioning across ICU4X major versions in this case.

Reply:
Also we never published a `CalendarWeekV1`. We had a `WeekDataV1`.

Reply:
r+ on CalendarWeekV1 then

All the hashes having changed is somewhat compelling to me. But you'd have to ask shane.

Reply:
all the other markers were renamed as well

Reply:
Ah, r+ then.